R/package_update.R

```diff
@@ -0,0 +1,140 @@
+#' Updates qPackages
```

Comment:
@EstherPeev did you check in qEnviron if this function works as expected? Thank you.

Reply:
Yes the function worked well on qEnviron as you can see from the last commit